--- auth_dovecot.c	2015-05-22 15:53:14.498661460 +0000
+++ auth_dovecot.c	2015-05-22 15:06:07.229929153 +0000
@@ -127,7 +127,9 @@
 void
 auth_init(int argc, char **argv, stralloc *login, stralloc *authdata)
 {
+#ifdef DEBUG
 	extern unsigned long loglevel;
+#endif
 	char		*l, *p;
 	unsigned int	uplen, u;
 	int		n, opt;
@@ -140,10 +142,12 @@
 		case 'd':
 			pbstool = optarg;
 			break;
+#ifdef DEBUG
 		case 'D':
 			scan_ulong(optarg, &loglevel);
 			loglevel &= ~256;	/* see auth_mod.c */
 			break;
+#endif
 		default:
 			auth_error(AUTH_CONF);
 		}
